### DISCUSSION TITLE
Fetch call Blocked (mixed-content)

### DISCUSSION THREAD

**Discussion by danboyce92** [vercel/community#3804] [Help]
### Question

I have an app up on vercel but I'm having trouble connecting the backend to it. I have the backend on a VPS on Azure running 24/7 but the requests don't come through because I don't have an SSL cert I think. Is there a way to get Vercel to ignore this?
This app is for my portfolio as a new dev I'm not really worried about security risks as nothing important is vulnerable.
Thanks for your time, any help would be greatly appreciated!
Daniel

**Comment by amyegan** [comment]
Hey @danboyce92. Do you see any extra info about the connection error in the [runtime logs](https://vercel.com/docs/observability/runtime-logs#) or in your [browser console](https://developer.chrome.com/docs/devtools/console/)?
